index.js

```diff
@@ -3,42 +3,44 @@ const BridgeTx = require("./BridgeTx");
 const BridgeMethod = require("./BridgeMethod");
 const BridgeEvent = require("./BridgeEvent");
 const utils = require("./utils");
+const {ethers} = require('ethers');
+const {formatBigIntForJson} = require("./utils");
 
 class BridgeTransactionParser {
 
-    constructor(web3Client) {
-        if (!web3Client) {
-            throw new Error(`web3Client is required`);
+    constructor(rskClient) {
```

Comment:
If a web3 client instance is passed here will it still work? If the answer is no then it's probably a major version update

Reply:
It won't work. It is mandatory to pass a ethersjs client instance(provider). 

I'll update the version.

Reply:
Done